app/models/task.rb

```diff
@@ -137,7 +142,8 @@ def result_details
   end
 
   def result_task_log
-    task_logs.where(status: status).order('logged_at ASC').last
+    logger.debug("result_task_log: status: #{status}")
+    task_logs.where(status: Task.statuses[status.to_s]).order('logged_at ASC').last
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.